public/api-docs/swaggerDoc.json

```diff
@@ -314,6 +314,45 @@
                 }
             }
         },
+        "/api/v1/:trip/comment": {
+            "post": {
+                "tags": [
+                    "Comment"
+                ],
+                "summary": "Make a Comment",
+                "description": "Allows a user to make a comment",
+                "operationId": "CreateComment",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "in": "body",
+                        "name": "comment",
+                        "schema": {
+                            "type": "object",
+                            "properties": {
+                                "comment": {
+                                    "type": "string",
+                                    "example": "Nice Location"
+                                }
+                            }
+                        }
+                    }
+                ],
+                "responses": {
+                    "201": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/User"
```

Comment:
you may want to create a new response definition as I don't think this `User defintions` tally with the response the api returns